CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ and this project adheres to
 
 ### Fixed
 
-## 0.11.3 2020-04-24
+## 0.11.4 2020-04-24
```

Comment:
I missed something, what happened to 11.1-3?

Reply:
I'm dumb and I wasn't tagging it correctly to publish.